tests/Functional/basic-data-table-incremental-fetch/config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "parameters": {
+    "name": "table1",
+    "outputTable": "table1",
+    "table": {
+      "schema": "testdb",
+      "tableName": "table1"
+    }
```

Comment:
If this is testing incremental fetching then I think `"incrementalFetchingColumn"` should be set here.
Also for incremental fetching it would be good to have a follow-up test that provides a state.json file with `lastFetchedRow` set

Reply:
You're right, I've done some refactoring yesterday and I swear I was doing that too. Don't know how could that happen.